### DISCUSSION TITLE
右手側の特定の列が動かない

### DISCUSSION THREAD

**Discussion by hirosat** [Yowkees/keyball#223] [Q&A] [answered]
始めまして！アドバイスいただけると助かります
<img width="495" alt="スクリーンショット 2023-05-01 13 42 24" src="https://user-images.githubusercontent.com/3906298/235408176-47704fdc-1f67-478d-a850-3786ee038e88.png">

上記の画像のように右手側の内側から３列目のキーが反応しない状況です。
まずはProMicroを疑い、左手側と右手側のProMicroを付け替えてみましたが、
同じ状況なので、ダイオードかキーソケットのはんだ状態を疑っています。
上記の症状の場合、具体的にどの場所で問題が起こりそうか特定することは可能でしょうか？
（一応テスターで問題列を縦に端子をあててみるとブザーがなったので、導通はしていそうに思えます）

**Comment by hirosat** [comment]
![PXL_20230501_045754486](https://user-images.githubusercontent.com/3906298/235409007-a1e3d7f6-cd17-4675-8f9d-1ef99082a353.jpg)
ご参考までに、問題のある基板の写真を取りましたので、このダイオード、とか印を付けていただけると助かります

**Comment by hirosat** [comment, marked as answer]
自己レスとなりますが、直りました。
22番の下の2つ並んでいるダイオードの下側の方が問題でした。
お騒がせしました

**Comment by Yowkees** [comment]
ご連絡が遅くなり申し訳ありませんでした。ご自身で解決いただけたとのことでありがとうございました。
